ui/components/app/advanced-gas-controls/advanced-gas-controls.component.js

```diff
@@ -0,0 +1,82 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18nContext } from '../../../contexts/i18n';
+import Typography from '../../ui/typography/typography';
+import {
+  FONT_WEIGHT,
+  TYPOGRAPHY,
+  COLORS,
+} from '../../../helpers/constants/design-system';
+import AdvancedGasControlsRow from './advanced-gas-controls-row.component';
+
+export default function AdvancedGasControls({
+  onGasLimitChange,
+  onMaxPriorityFeeChange,
+  onMaxFeeChange,
+}) {
+  const t = useContext(I18nContext);
+
+  return (
+    <div className="advanced-gas-controls">
+      <AdvancedGasControlsRow
+        titleText={t('gasLimit')}
+        onChange={onGasLimitChange}
+        tooltipText=""
+        titleDetailText=""
+      />
+      <AdvancedGasControlsRow
```

Comment:
[REQ] Numeric input requires a value prop to be passed in as it's a controlled input. These will eventually be grabbed from a hook so for the time being I think we could be good to just use a useState hook for these values/setters so that we have working components for the time being and then we can replace the values/setters with more appropriate hooks in a follow up PR once the data is available.